source/c.txt

```diff
@@ -17,12 +17,11 @@ MongoDB C Driver
 Introduction
 ------------
 
-The MongoDB C Driver, also known as “libmongoc”, is the official client library
-for C applications, and provides a base for MongoDB drivers in higher-level
-languages.
-
-The library is compatible with all major platforms. It depends on libbson
-to create and parse BSON data.
+Welcome to the documentation site for the official MongoDB C driver. 
+You can add the driver to your application to work with MongoDB in C. 
+Download the required libraries, ``libmongoc`` and ``libbson``,  from 
+`mongoc.org <http://mongoc.org/libmongoc/current/installing.html>`__ or 
```

Comment:
I tried changing the http -> https and it looks like the certificates are in place for the domain (so you should change it to https).

Reply:
Good call